### PR TITLE
Feature highlighter

### PR DESCRIPTION
**Pull request by inbalwix**
Adding native adjustments

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -36,6 +36,9 @@ android {
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
+        debug {
```
Comment (ethanshar) [inline android/app/build.gradle]: what's that?

File: demo/src/screens/componentScreens/FeatureHighlightScreen.js [hunk 2]
```diff
@@ -7,47 +8,84 @@ class FeatureHighlightScreen extends Component {
     super(props);
     this.closeHighlight = this.closeHighlight.bind(this);
     this.showHighlight = this.showHighlight.bind(this);
+    this.onRefUpdated = this.onRefUpdated.bind(this);
+
+    this.state = {
+      showFTE: false,
+      fteHighlightViewTag: undefined,
```
Comment (ethanshar) [inline demo/src/screens/componentScreens/FeatureHighlightScreen.js]: what is fteHighlightViewTag for, are you using it anywhere?

File: android/app/build.gradle (hunk 1 above)
Comment (inbalwix) [reply android/app/build.gradle]: to be able to debug with breakpoints

File: demo/src/screens/componentScreens/FeatureHighlightScreen.js (hunk 2 above)
Comment (inbalwix) [reply demo/src/screens/componentScreens/FeatureHighlightScreen.js]: No, you're right. This is leftover from Artal's fix suggestion. I will remove it.

**Pull request merged**